master/getting-started/kubernetes/installation/hosted/calico.yaml

```diff
@@ -21,6 +21,9 @@ data:
   # Configure the Calico backend to use.
   calico_backend: "bird"
 
+  # Configure the MTU to use
+  cni_mtu: "1500"
```

Comment:
I think we just want to call this "mtu" or perhaps "veth_mtu" since it's used by both CNI and Felix.

Reply:
Also, we should default to the lowest common denominator - from [this page](https://docs.projectcalico.org/,latest/usage/configuration/mtu) it looks like that's 1440.

It would also be great to update the command above with a link for more information.

e.g.:
"# Configure the MTU to use for container veths. See here for more information: https://docs.projectcalico.org/v3.0/usage/configuration/mtu"